### DISCUSSION TITLE
cant not support img file?

### DISCUSSION THREAD

**Discussion by 0zcl** [handlebars-lang/handlebars.js#1781] [Q&A]
Before filing issues, please check the following points first:

- [ ] Please don't open issues for security issues. Instead, file a report at https://www.npmjs.com/advisories/report?package=handlebars
- [ ] Have a look at https://github.com/wycats/handlebars.js/blob/master/CONTRIBUTING.md
- [ ] Read the FAQ at https://github.com/wycats/handlebars.js/blob/master/FAQ.md
- [ ] Use the jsfiddle-template at https://jsfiddle.net/4nbwjaqz/4/ to reproduce problems or bugs

This will probably help you to get a solution faster. 
For bugs, it would be great to have a PR with a failing test-case.

I use metalsmith to transer file from sourceName dir to destination dir. and .png file include in sourceName dir. 
and then I use handles.comile。but error happen。can anyone help me?
![image](https://user-images.githubusercontent.com/26007970/103075790-4d44c500-4607-11eb-9e51-f0619817997d.png)

**Comment by jaylinski** [comment]
@0zcl Can you provide a reduced test-example? https://jsfiddle.net/9D88g/180/
